content/blog/getting-started-with-hpe-greenlake-api-for-backup-and-recovery.md

```diff
@@ -0,0 +1,591 @@
+
+   ```shellsession
+   GET /backup-recovery/v1beta1/protection-jobs?filter=assetInfo/id eq {{vmId}}&select=assetInfo,id,operational,protections
+   ```
+
+![API to figure out protection-jobs](/img/api-to-figure-out-protection-jobs.png)
+
+2. This use case is equivalent to run now button at the schedule of the `“0-Linux-Demo-V02”` on the selected cloud protection as shown in below figure. After clicking `“Run Now”` button, a cloud protection will commence against the virtual machine, a recovery point is going to be created at the cloud protection store. On next step, I will invoke `"Run Now"` on the `HPE Cloud Protection Store` using HPE GreenLake API for Backup and Recovery.
+
+![UI for run now protection jobs](/img/gui-run-now-cloud-protection.png)
+
+3. The GreenLake API to accomplish the use case above was `POST /backup-recovery/v1beta1/protection-jobs/:id/run`, and the documentation of this [API](https://developer.greenlake.hpe.com/docs/greenlake/services/backup-recovery/public/openapi/backup-recovery-public-v1beta1/operation/DataManagementJobRun) also list the required JSON structure for the request body. I created a JSON request body structure as the following:
+
+   ```jsonc
+   {
+      "﻿fullBackup": false,
+      "﻿ScheduleIds": [3]
+   }
+   ```
+
+> **Note** that there was a key called `“fullBackup”` inside the JSON request body to enable the creation of full protection where a backup will be created independently from the existing copies in the protection store. I also entered number 3 into `ScheduleIds` JSON array, to represent the cloud backup schedule. The below figure shows an example of the execution of `"run now"` without full backup protection of the third schedule which is cloud protection of this virtual machine. The value `<protection-jobs-id>` will be entered from the parameter of this API in this manner: `POST /backup-recovery/v1beta1/protection-jobs/"<protection-jobs-id>"/run`.
+
+![API to execute a protection run](/img/api-to-execute-a-protection.png)
+
+4. The result from the API execution above can be validated from the API `/data-services/v1beta1/async-operations/:id` using the task id obtained from the above response header. The API used for this: `G﻿ET /data-services/v1beta1/async-operations/{{taskId}}`.
+
+![API async-operations of execution of protection-jobs](/img/api-task-list-after-a-run-execution.png)
+
+The activities above were validated from the HPE GreenLake Backup and Recovery list of the recovery points as shown in the below figure.
+
+![GUI display the completed cloud protection run](/img/gui-cloud-run-now-completed-succesfully.png)
+
+## Wow.. those were so cool! Can I recover a new virtual machine from that recovery point that I just created?
+
+Each of the recovery points regardless of the location of store (array snapshot, On-Premises Protection-Store, or HPE Cloud Protection-Store) can be recovered using this GreenLake API: `POST /backup-recovery/v1beta1/virtual-machines/:id/restore`.
```

Comment:
Place commas around this phrase "regardless of the location of store (array snapshot, On-Premises Protection-Store, or HPE Cloud Protection-Store)"

Add "HPE" in front of "GreenLake API"